README.md

```diff
@@ -18,7 +20,7 @@ Or install it yourself as:
 
     $ gem install graphql_devise
 
-GraphQL interface on top of the [Devise Token Auth](https://github.com/lynndylanhurley/devise_token_auth) (DTA) gem.
+GraphQL's interface is on top of the [Devise Token Auth](https://github.com/lynndylanhurley/devise_token_auth) (DTA) gem.
```

Comment:
The original text was correct, but as I mentioned in the last comment, this should be moved to line 5

Reply:
Ok I'll go with the original text. I think it might be more clear/grammatically correct to say "A GraphQL interface...". WDYT?

Reply:
I think as that is the gem's short description of what it is, I think original text works better although yours is also grammatically correct